api/src/pcapi/core/offerers/repository.py

```diff
@@ -138,8 +138,8 @@ def get_offer_counts_by_venue(venue_ids: Iterable[int]) -> dict[int, int]:
     return offers_count_by_venue_id
 
 
-def get_venues_with_offers(offererId: int) -> Iterable[int] | None:
-    """Return a dictionary with ids of venues with non-draft offers.
+def get_venues_with_offers(offererId: int) -> Iterable[int]:
+    """Return an array with ids of venues with non-draft offers.
```

Comment:
(détail) Ici, c'est ~Paris~ Python. On n'a pas de `Array`, mais on a `list`. ;)

```suggestion
    """Return a list with ids of venues with non-draft offers.
```